test/membrane_live/webinars_test.exs

```diff
@@ -55,13 +58,15 @@ defmodule MembraneLive.WebinarsTest do
       update_attrs = %{
         "description" => "some updated description",
         "presenters" => [],
+        "moderators" => [],
         "start_date" => ~N[2022-07-18 10:20:00],
         "title" => "some updated title"
       }
 
       assert {:ok, %Webinar{} = webinar} = Webinars.update_webinar(webinar, update_attrs)
       assert webinar.description == "some updated description"
       assert webinar.presenters == []
+      assert webinar.moderators == []
```

Comment:
Shouldn't there be always one email, the email of the creator of the event?

Reply:
You're right, we should never remove the owner from moderators.